client/state/email-accounts/reducer.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Internal dependencies
+ */
+import {
+	combineReducers,
+	keyedReducer,
+	withSchemaValidation,
+	withoutPersistence,
+	withStorageKey,
+} from 'state/utils';
+import {
+	EMAIL_ACCOUNTS_REQUEST,
+	EMAIL_ACCOUNTS_REQUEST_FAILURE,
+	EMAIL_ACCOUNTS_REQUEST_SUCCESS,
+} from 'state/action-types';
+import { usersSchema } from './schema';
+
+export const usersReducer = withSchemaValidation( usersSchema, ( state = null, action ) => {
+	switch ( action.type ) {
+		case EMAIL_ACCOUNTS_REQUEST_FAILURE:
+			return null;
+
+		case EMAIL_ACCOUNTS_REQUEST_SUCCESS: {
+			const {
+				response: { accounts },
+			} = action;
+
+			return accounts;
```

Comment:
Do we really need to destructure the list of accounts like this? What do you think of the following approach?

```javascript
return action.response.accounts;
```

Reply:
You've marked it as resolved but I still see the old code? If you don't agree with me, which is perfectly fine, would you mind at least explaining your point-of-view? That's how we can learn from each other 🙂.